examples/lock-app/nrfconnect/main/ZclCallbacks.cpp

```diff
@@ -126,9 +126,5 @@ void emberAfDoorLockClusterInitCallback(EndpointId endpoint)
     logOnFailure(DoorLock::Attributes::NumberOfCredentialsSupportedPerUser::Set(endpoint, CONFIG_LOCK_NUM_CREDENTIALS_PER_USER),
                  "number of credentials per user");
 
-    // Set FeatureMap to (kUser|kPinCredential), default is:
-    // (kUser|kAccessSchedules|kRfidCredential|kPinCredential) 0x113
-    logOnFailure(DoorLock::Attributes::FeatureMap::Set(endpoint, 0x101), "feature map");
```

Comment:
This was needed because in the upstream we have shared lock-app.zap which used a different feature set. Do you think it should be removed in the upstream?

Reply:
Right, I brought it back, but changed value to 0x181 to match the NCS sample.